README.md

```diff
@@ -60,6 +63,30 @@ Please note: only JSON format is supported currently.
 
 ![Extract](resources/extract.gif)
 
+## Import Function App to create a new API
+![importFuncApp](resources/importFunc.gif)
+
+## Import Function App to an existing API
+![importFuncAppToApi](resources/importFuncToApi2.gif)
+
+## Import Web App (with swagger object) to create a new API

```

Comment:
dont need all three. Just the happy path where swagger exists and leave other to documentation.